### ISSUE TITLE
Abbilty to specify custom path for helm charts and tests

### ISSUE THREAD

**Issue by AiRrRock** [kiwigrid/helm-gradle-plugin#25]
It will be nice to add the ability to specify custom path for the helm chrarts and test. This is needed to use plugin with more complex projects.

**Comment by wind57** [comment]
@jekkel sorry if this is obvious, just a question. This is about where a chart should end-up, right? directory wise. Cause the source of a chart can be easely altered via : 


```
tasks.withType<HelmBuildTask>().configureEach {
    source.set(file(<SOME_DIFF_DIRECTORY>))
}
```

**Comment by jekkel** [comment]
@wind57 No, it is actually about the sources (although the final directory can be modified in a similar easy way with the property `outputDirectory`). And yes you're right it's already possible. But it is not possible to change the source folder of the **test** task, i.e. the folder which contains the test cases and test yaml files. The PR is going to add it and unifies handling a bit on top of that.
